fulp_modules/main_features/bloodsuckers/code/antagonists/bloodsucker_datum.dm

```diff
@@ -344,8 +345,24 @@
 	feed_amount += 2 // Increase how quickly I munch down vics (15)
 	max_blood_volume += 100 // Increase my max blood (600)
 	/// Assign True Reputation
-	if(bloodsucker_level == 4)
+	if(bloodsucker_level == 3)
 		SelectReputation(am_fledgling = FALSE, forced = TRUE)
+		var/chosen_clan = pick(
+			CLAN_BRUJAH,
+			CLAN_NOSFERATU,
+			CLAN_TREMERE,
+			CLAN_VENTRUE,
+			)
+		switch(chosen_clan)
+			if(CLAN_BRUJAH)
+				my_clan = CLAN_BRUJAH
+			if(CLAN_NOSFERATU)
+				my_clan = CLAN_NOSFERATU
+			if(CLAN_TREMERE)
+				my_clan = CLAN_TREMERE
+			if(CLAN_VENTRUE)
+				my_clan = CLAN_VENTRUE
```

Comment:
Can't you just:
```suggestion
		my_clan = pick(
			CLAN_BRUJAH,
			CLAN_NOSFERATU,
			CLAN_TREMERE,
			CLAN_VENTRUE,
			)
```
Unless you are planning to do epic stuff for each clan in here.

Reply:
I originally did but moved it to the assignrandombane proc itself